deploy_pulp3/pulp3/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+pulp_user: pulp
+pulp_venv : "/home/pulp/pulpvenv"
```

Comment:
If someone overrides `pulp_user`, then do they also have to override `pulp_venv`? This seems like something that can be derived with the getent module.

Reply:
as far as i understood we will stick to pulp user instead of apache.

Reply:
And? If a user is allowed to change under which user Pulp is installed, let's do it well.